lib/matplotlib/backend_tools.py

```diff
@@ -750,6 +750,9 @@ def disable(self, event):
     def trigger(self, sender, event, data=None):
         self.toolmanager.get_tool(_views_positions).add_figure(self.figure)
         super().trigger(sender, event, data)
+        new_navigate_mode = self.name.upper() if self.toggled else None
+        for ax in self.figure.axes:
```

Comment:
CodeCov isn't wrong here, this code is not executed by any of the tests, including the one added below.

Reply:
Yes, it is not. Thank you. I have made few changes to fix this.